pkg/controller/jaeger/jaeger_controller.go

```diff
@@ -21,9 +22,14 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/source"
 
 	"github.com/jaegertracing/jaeger-operator/pkg/apis/jaegertracing/v1"
+	"github.com/jaegertracing/jaeger-operator/pkg/inject"
 	"github.com/jaegertracing/jaeger-operator/pkg/strategy"
 )
 
+const (
+	finalizer = "uninstall-jaeger"
```

Comment:
shouldn't we use something like `finalizer.jaegertarcing.io`?

Reply:
Yes, better name! thanks!

Reply:
+1! Name changed!